app/graphql/types/list_type.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Types
+  class ListType < Base::Object
+    I18N_PATH = 'graphql.types.list_type'
+
+    graphql_name 'ListType'
+    implements Types::Interfaces::Node
+
+    description I18n.t("#{I18N_PATH}.desc")
+
+    field :name,
+          String,
+          null: false,
+          description: I18n.t("#{I18N_PATH}.fields.name")
+
+    field :items,
+          [Types::MovieType],
+          null: true,
```

Comment:
`null: false`. If there are no items this field will return an empty array, not `null`